### PR TITLE
chore: gha to check release file diff

### PR DESCRIPTION
**Pull request by josecorella**
_Issue #, if available:_

_Description of changes:_
This workflow checks if this pull request is modifying either `*release.yml` files or the `check-files.yml` file. 
This allows us to control and set a minimum number of approvals (2) from CODEOWNERS if these files are being modified.

_Squash/merge commit message, if applicable:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ajewellamz — DISMISSED** [review]

**Review by lucasmcdonald3 — DISMISSED** [review]
Approving so we can test what happens once we get 2 approvals, will approve again once we've validated this

**Review by lucasmcdonald3 — APPROVED** [review]

**Review by ajewellamz — APPROVED** [review]

**Pull request merged**
